### ISSUE TITLE
Can't Handle "Plain" Messages Mixed In

### ISSUE THREAD

**Issue by DakotaNelson** [DakotaNelson/sneaky-creeper#73]
From the new tests:

```
======================================================================
ERROR: Test the twitter channel with the full range of printable characters.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/dnelson/projects/sneaky-creeper/venv/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/dnelson/projects/sneaky-creeper/sneakers/tests/test_all.py", line 33, in unit_channel
    got = t.receive()
  File "/home/dnelson/projects/sneaky-creeper/sneakers/exfil.py", line 154, in receive
    packet_total = base94.decode(str(tokenized[1]))
IndexError: list index out of range
```

This is happening because some of the tweets (the ones generated by the Twitter tests module) are just a few characters, such as `CWDMazjpvW` - not in the format sneaky-creeper expects. When it can't tokenize the message based off spaces, it gets scared and dies.

We should fix this so that it simply ignores improperly formatted messages and moves on.

**Comment by davinerd** [comment]
Not sure if ignoring that messages is a good solution.
In this case we'll break one of the library's main feature.

But we need a clever solution, as quickly.

**Comment by DakotaNelson** [comment]
So, imagine a Twitter stream from a user that has these 4 tweets:

a b sometextorsomething

some other tweet #twittering 

b c this is the last part of a long message

a c this is the first part of a long message

Right now, screep fails on the second tweet down - "some other tweet" - because it tries to split it and read the header, and fails (and crashes). It makes sense to ignore this tweet (since we obviously didn't create it and therefore it isn't a sneaky-creeper message) so that other content can be mixed in with the sneaky-creeper accounts - both for (theoretically) camouflage and just so that it's resistant to junk.

You're right, though - this is a  pretty high-priority issue.

**Comment by davinerd** [comment]
Fixed here, it should work: 8359795

**Comment by DakotaNelson** [comment]
(Do note that this will still likely fail on any message with 3 or more space-separated phrases, so we have likely only delayed the problem. Will re-open when this reoccurs.)
